staticDHCPd/staticdhcpdlib/config.py

```diff
@@ -210,6 +212,19 @@
         handleUnknownMAC = conf.handleUnknownMAC
 else:
     handleUnknownMAC = lambda *args, **kwargs : None
+
+if hasattr(conf, 'filterRetrievedDefinitions'):
+    filterRetrievedDefinitions = conf.filterRetrievedDefinitions
+else:
+    def filterRetrievedDefinitions(definitions, *args, **kwargs):
+        if not definitions:
+            return None
```

Comment:
If you make that change you should also change this: https://github.com/Navisite/staticdhcpd/pull/5/files#diff-4f4c9f9e5012de9b7f6da0369cc35257R26